web/templates/layouts/WikiLayout.tpl

```diff
@@ -48,7 +48,7 @@
 
  	<meta http-equiv="content-type" content="text/html;charset=UTF-8"/>
     <meta http-equiv="content-language" content="{$site->getLanguage()}"/>
-
+    <meta name="csrf-token" content="{php}echo(csrf_token());{/php}"/>
```

Comment:
Is `{php}...{/php}` different from `{...}`?

Reply:
It seems to be, the same command that way threw an error, I believe that syntax is just for variables that were passed to the template.